content/70.coi-contribs.md

```diff
@@ -11,3 +11,8 @@
 |Author|Contributions|
 |---|---|{% for author in manubot.authors %}
 |{{author.name}}|{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}|{% endfor %}
+
+### Acknowledgements
+
+We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
```

Comment:
```suggestion
We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
We are grateful to the following contributors for reviewing pieces of the text: Ipsita Krishnan.
```
This list will be much longer than just one person, hence the weird formatting for now.